src/Goerli-DssSpellCollateral.sol

```diff
@@ -17,8 +17,66 @@
 pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExecLib.sol";
+import "dss-interfaces/dss/VatAbstract.sol";
+import "dss-interfaces/dapp/DSPauseAbstract.sol";
+import "dss-interfaces/dss/JugAbstract.sol";
+import "dss-interfaces/dss/SpotAbstract.sol";
+import "dss-interfaces/dss/GemJoinAbstract.sol";
+import "dss-interfaces/dapp/DSTokenAbstract.sol";
+import "dss-interfaces/dss/ChainlogAbstract.sol";
+import "dss-interfaces/dss/IlkRegistryAbstract.sol";
+
+interface ERC20Like {
+    function approve(address, uint256) external returns (bool);
+}
+
+interface RwaLiquidationLike {
+    function wards(address) external returns (uint256);
```

Comment:
Cleanup: consider checking and removing unused interfaces

Reply:
`wards` is not used in the current spell code, consider removing it and check other unused interfaces, keeping only the ones that are used to reduce code size and ease review.

Reply:
Honestly I don't see a point in doing that, because the interfaces for such components are kind of immutable. They shouldn't be a part of the review.

This could be put into `dss-interfaces` like the other `*Abstract` contracts, moved to another file or even moved to the end of the file.

Reply:
I created a [PR](https://github.com/makerdao/dss-interfaces/pull/75) to add the MIP21 contracts to `dss-interfaces`.